it/controllers/RegularLengthEmployedControllerISpec.scala

```diff
@@ -0,0 +1,296 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import assets.BaseITConstants
+import assets.PageTitles._
+import config.featureSwitch.{ExtensionTwoNewStarterFlow, FeatureSwitching}
+import models.PaymentFrequency.Monthly
+import models.{RegularLengthEmployed, UserAnswers}
+import play.api.http.Status._
+import play.api.libs.json.Json
+import utils.{CreateRequestHelper, CustomMatchers, ITCoreTestData, IntegrationSpecBase}
+
+class RegularLengthEmployedControllerISpec extends IntegrationSpecBase
+  with CreateRequestHelper with CustomMatchers with BaseITConstants with ITCoreTestData with FeatureSwitching {
+
+  "the ExtensionTwoNewStarterFlow is turned ON" when {
+
+    "GET /regular-length-employed" should {
+
+      "redirect display the correct title" in {
+
+        enable(ExtensionTwoNewStarterFlow)
+
+        val userAnswers: UserAnswers =
+          emptyUserAnswers
+            .withClaimPeriodStart("2020, 11, 1")
+            .withClaimPeriodEnd("2020, 11, 30")
+            .withFurloughStartDate("2020, 11, 1")
+            .withFurloughStatus()
+            .withPaymentFrequency(Monthly)
+            .withNiCategory()
+            .withPensionStatus()
+            .withPayMethod()
+            .withLastPayDate("2020, 10, 31")
+            .withPayDate(List("2020, 10, 31"))
```

Comment:
note: I am assuming these are the mandatory answers needed just adjusted the claim start dates and stuff needed for this pages nav logic

Reply:
they have a method which builds monthly mandatory answers just based it off that
`  def mandatoryAnswersOnRegularMonthly =
    emptyUserAnswers
      .withClaimPeriodStart("2020, 3, 1")
      .withClaimPeriodEnd("2020, 3, 31")
      .withFurloughStartDate("2020, 3, 1")
      .withFurloughStatus()
      .withPaymentFrequency(Monthly)
      .withNiCategory()
      .withPensionStatus()
      .withPayMethod()
      .withLastPayDate("2020, 3, 31")
      .withPayDate(List("2020, 2, 29", "2020, 3, 31"))`